collector/__init__.py

```diff
@@ -447,18 +426,18 @@ def _process_observation_data(self,
                         # Timing window starts at the beginning of the sequence, the slew can be outside the window
                         # Therefore, subtract acquisition time from start of timing window
                         wstart -= self.observations[-1].acquisition()
-                        windows.append(Time([roundMin(wstart), roundMin(wend)]))
+                        windows.append(Time([round_min(wstart), round_min(wend)]))
                 self.obs_windows.append(windows)
                 
                 # Observation number in program
-                collected_program.add(self.nobs)
+                collected_program.add_observation(self.observation_num)
 
 
                 # Check if group exists, add if not
                 if group['key'] not in self.scheduling_groups.keys():
                     self.scheduling_groups[group['key']] = {'name': group['name'], 'idx': []}
-                    collected_program.add(group['key'])
-                self.scheduling_groups[group['key']]['idx'].append(self.nobs)
+                    collected_program.add_group(group['key'])
+                self.scheduling_groups[group['key']]['idx'].append(Collector.observation_num)
```

Comment:
Why here is `Collector.` but above in 433 is `self.` being used?

Reply:
Ahhh... so my understanding is that `nobs` was a placeholder for `number_of_observations`.
We can make this a static member instead of a `Collector` implementation specific number, in which case, we **can** access it through `self`, but that's simply deceptive because `observation_num` now belongs to all `Collector` instances.

Do you foresee any problems with that? I couldn't, but I still have lots of reading to do. :-)